operator/src/main/java/org/bf2/operator/controllers/ManagedKafkaController.java

```diff
@@ -128,10 +121,6 @@ public UpdateControl<ManagedKafka> reconcile(ManagedKafka managedKafka, Context
         }
     }
 
-    @Override
-    public List<EventSource> prepareEventSources(EventSourceContext<HasMetadata> context) {
-        return Arrays.asList(eventSource);
-    }
```

Comment:
This and the `EventSourceInitializer<HasMetadata>` interface are still needed. It looks like the return type changed to `Map<String, EventSource>` so the method body could instead be something like `return Map.of("ownedResources", eventSource)`

Reply:
Added this back!